src/Evolution/Initialization/Initialize.hpp

```diff
@@ -54,9 +56,18 @@ struct RemoveOptionsAndTerminatePhase {
                     const ArrayIndex& /*array_index*/, ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
     using options_tags_to_remove = option_tags<ActionList>;
+    // Retrieve the `initialization_option_tags` that are still in the DataBox
+    // and remove them.
     using options_tags_to_remove_this_time = tmpl::filter<
         options_tags_to_remove,
-        tmpl::bind<tmpl::list_contains, tmpl::pin<DbTagsList>, tmpl::_1>>;
+        tmpl::bind<
+            tmpl::list_contains,
+            tmpl::pin<typename db::DataBox<DbTagsList>::simple_item_tags>,
+            tmpl::_1>>;
+    static_assert(std::is_same<tmpl::back<ActionList>,
+                               RemoveOptionsAndTerminatePhase>::value,
+                  "The last action in the initialization phase must be "
+                  "Initialization::Actions::RemoveOptionsAndTerminatePhase.");
```

Comment:
It's the other way round: (If using this action,) `Initialization::Actions::RemoveOptionsAndTerminatePhase must be the last action in the initialization phase`. If the user doesn't use this action, e.g. removes option tags somewhere else, then some other action would end the initialization phase. To be clear, I'm only nitpicking on the grammar of this error message, not the functionality.

Reply:
I literally don't understand what your saying. I'm ignoring this.